SU-HSDO/suhumsci
# March 11 fixes

# READY FOR REVIEW

# Summary
- Added humanitiescore multisite
- Switched the link button in the wysiwyg for the one from core (similiar to what we did for the D8CORE)
- Fixed a generic.png file missing on existing sites for media files in the media library.
- Fixed composer dependency issues and failing tests.
- added a post multisite hook for blt similar to what i implemented for ace-gryphon that modifies the drush alias and the blt.yml files

# Need Review By (Date)
- ASAP please

# Urgency
- high

# Steps to Test
1. checkout this branch
1. `composer install`
1. install a new site `blt drupa:install`
1. add a page with a text area, add an image to the wysiwyg and verify you can add a link to that media item.
1. review code for the new multisite/blt commands/drush aliases.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
